Yosemite/YosemiteTests/Model/Extensions/PaymentIntent+ReceiptParametersTests.swift

```diff
@@ -2,37 +2,47 @@ import XCTest
 @testable import Yosemite
 
 final class PaymentIntent_ReceiptParametersTests: XCTestCase {
-    func test_receipt_parameters_is_generated_from_valid_intent() {
-        let intent = MockPaymentIntent.mock()
+    func test_receipt_parameters_is_generated_from_intent_with_a_charge_that_contains_paymentMethod_card_details() {
+        // Given
+        let intent = PaymentIntent.fake().copy(amount: 100, charges: [.fake().copy(paymentMethod: .cardPresent(details: .fake()))])
 
+        // When
         let receiptParameters = intent.receiptParameters()
 
+        // Then
         XCTAssertEqual(receiptParameters?.amount, intent.amount)
         XCTAssertEqual(receiptParameters?.currency, intent.currency)
 
         guard let cardDetails = intent.charges.first?.paymentMethod?.cardPresentDetails else {
-            XCTFail()
-            return
+            return XCTFail()
         }
 
         XCTAssertEqual(receiptParameters?.cardDetails, cardDetails)
     }
 
     func test_receipt_parameters_includes_store_from_intent_metadata() {
-        let intent = MockPaymentIntent.mock()
+        // Given
+        let metadata = PaymentIntent.initMetadata(store: "Store Name", orderID: 134)
+        let intent = PaymentIntent.fake().copy(metadata: metadata, charges: [.fake().copy(paymentMethod: .cardPresent(details: .fake()))])
 
+        // Then
         XCTAssertEqual(intent.receiptParameters()?.storeName, "Store Name")
     }
 
     func test_receipt_parameters_includes_orderID_from_intent_metadata() {
-        let intent = MockPaymentIntent.mock()
+        // Given
+        let metadata = PaymentIntent.initMetadata(store: "Store Name", orderID: 1920)
+        let intent = PaymentIntent.fake().copy(metadata: metadata, charges: [.fake().copy(paymentMethod: .cardPresent(details: .fake()))])
 
+        // Then
         XCTAssertEqual(intent.receiptParameters()?.orderID, 1920)
     }
 
     func test_receiptParameters_Includes_formattedAmount_WithDecimalFormatting() {
-        let intent = MockPaymentIntent.mock()
+        // Given
+        let intent = PaymentIntent.fake().copy(amount: 10000, charges: [.fake().copy(paymentMethod: .cardPresent(details: .fake()))])
```

Comment:
Since the `charges` in these tests is not the main part of the test data (just needed to say "this is a card-present payment so should have a receipt") I think it would make the tests easier to read if it were pulled out into a `TestData` enum or similar, like we do with Localisation often.

```
enum TestData {
    static let cardPresentCharge = Charge.fake().copy(paymentMethod: .cardPresent(details: .fake()))
}
```

Reply:
Makes sense! I added an enum`Mocks` like how we did in other test cases in https://github.com/woocommerce/woocommerce-ios/pull/6817/commits/ad5326d54f3573d792c31bfb0e5bf02ebe712bc7